RELEASENOTES.md

```diff
@@ -7,6 +7,26 @@
 * Ubuntu 18.04 and 20.04 LTS
 * CentOS 7 and 8
 
+# v1.3.1 release notes
+
+This release requires [Libfabric v1.11.0](https://github.com/ofiwg/libfabric/releases/tag/v1.11.0)
+or later and supports [NCCL v2.12.12](https://github.com/NVIDIA/nccl/releases/tag/v2.12.12-1) while
+maintaining backward compatibility with older NCCL versions (up to [NCCL v2.4.8](https://github.com/NVIDIA/nccl/releases/tag/v2.4.8-1)).
+It was tested with Libfabric versions up to [Libfabric v1.14.0](https://github.com/ofiwg/libfabric/releases/tag/v1.14.0).
+
+Bug Fixes:
+* Fix compilation on CentOS 7
+* Update Control Message Tag Bits
+* Fix reuse check in ofi_iaccept
```

Comment:
Rename this to `Fix the active connection issue for non-blocking accepts (impacts NCCL versions 2.12 and above)`.